x-pack/plugins/alerting/server/plugin.test.ts

```diff
@@ -121,61 +114,6 @@ describe('Alerting Plugin', () => {
       });
     });
 
-    it(`applies the default config if there is no rule type specific config `, async () => {
```

Comment:
forgot to ask this in the last review - feels like we'd still want to test this from the config standpoint, even if we have more elaborate tests for the variations elsewhere.  Always good to see us test all the possible config values, just to make sure the config code itself is right.  Which I think means this code should have been in `config.test.ts` anyway, not here.  And that we'd add new ones in `config.test.ts`.  Could be I missed another testing pass on this though.  And if we could use some more tests like that, I'd be fine doing it as a follow-on issue.

Reply:
I removed this because this is not happening in the plugin anymore.
The new config is tested in both: create_execution_handler.test.ts and get_actions_config_map.test.ts :)